.ci/idea-format.bat

```diff
@@ -0,0 +1,55 @@
+@echo off
+
+::----------------------------------------------------------------------
+:: IntelliJ IDEA inspections for checkstyle.
+::
+:: Example:
+:: SET IDEA_PATH=C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2017.2.1\bin\idea.bat
+:: .ci\idea-inspection.bat
+::----------------------------------------------------------------------
+
+SET PROJECT_DIR=%CD%
+SET FORMAT_PATH=%CD%\config\intellij-idea-code-style.xml
+SET RESULTS_DIR=%CD%\target\format-results
+SET IDEA_LOCATION=
```

Comment:
```suggestion
SET IDEA_LOCATION
```
does it work without the equal sign `=` too? dont know the syntax sorry.
https://ss64.com/nt/set.html
<img width="805" alt="image" src="https://github.com/user-attachments/assets/ea31e833-3b84-42f8-9c58-af9c9673bb4b" />

Reply:
lets keep `=`, it is more explicit